benchmark/king2dcompressible/ala/ala.prm

```diff
@@ -0,0 +1,286 @@
+#    set Function expression = if(z<z1,0.5+0.5*(z1-z)/z1,if(z>z2,0.5*(1-z)/(1-z2),0.5)) + 0.1 * cos(x*pi) * sin(z*pi)
+#    set Variable names      = x,z
+  end
+end
+
+
+# Then follows a section that describes the boundary conditions
+# for the temperature. The model we choose is called 'box' and
+# allows to set a constant temperature on each of the four sides
+# of the box geometry. In our case, we choose something that is
+# heated from below and cooled from above. (As will be seen
+# in the next section, the actual temperature prescribed here
+# at the left and right does not matter.)
+subsection Boundary temperature model
+  set Model name = box
+
+
+  subsection Box
+    set Top temperature    = 0.091
+    set Bottom temperature = 1.091
+    set Left temperature   = 0
+    set Right temperature  = 0
+  end
+end
+
+# We then also have to prescribe several other parts of the model such as
+# which boundaries actually carry a prescribed boundary temperature, whereas
+# all other parts of the boundary are insulated (i.e., no heat flux through
+# these boundaries; this is also often used to specify symmetry boundaries).
+subsection Model settings
+  set Fixed temperature boundary indicators   = top, bottom
+
+  # The next parameters then describe on which parts of the
+  # boundary we prescribe a zero or nonzero velocity and
+  # on which parts the flow is allowed to be tangential.
+  # Here, all four sides of the box allow tangential
+  # unrestricted flow but with a zero normal component:
+  set Zero velocity boundary indicators       = 
+  set Prescribed velocity boundary indicators =
+  set Tangential velocity boundary indicators = left, right, top, bottom
+
+  # The final part of this section describes whether we
+  # want to include adiabatic heating (from a small
+  # compressibility of the medium) or from shear friction,
+  # as well as the rate of internal heating. We do not
+  # want to use any of these options here:
+  set Include adiabatic heating               = false
+  set Include shear heating                   = false
+end
+
+subsection Heating model
+  # shear heating: 1/2 eta*e(u): ... need  -> 1/2 e(u): ..  * Di/Ra
+  # adiabatic heating: alpha T (u . \nabla p)
+  # 
+
+  set List of model names = adiabatic heating, shear heating#, constant heating
+
+  subsection Constant heating
+    set Radiogenic heating rate = 1.25e-12#5e-9
+  end
+
+  subsection Adiabatic heating
+    # A flag indicating whether the adiabatic heating should be simplified
+    # from $\alpha T (\mathbf u \cdot \nabla p)$ 
+    # to $ \alpha \rho T (\mathbf u \cdot \mathbf g) $. alpha=thermal exp
+
+    # have \alpha rho T w
+    # need: \alpha \rho T w Di
+    set Use simplified adiabatic heating = true
+  end
+end
+
+subsection Formulation
+  set Formulation = ALA # equivalent to 'custom' with the settings below
+  #set Mass conservation    = reference density profile
+  #set Temperature equation = reference density profile
```

Comment:
Do either, but not both: choose ALA and remove the two other lines, or select "custom" and leave them in. Otherwise, I think it is just too confusing; if you want to discuss the equivalence, the manual is a better place than a half-line comment in a .prm file.